tests/data_context/test_data_context_datasources.py

```diff
@@ -185,7 +189,7 @@ def test_get_datasource_cache_miss(
 @pytest.mark.unit
 def test_DataContext_add_datasource_updates_cache_and_store(
     cloud_data_context_in_cloud_mode_with_datasource_pandas_engine: DataContext,
-    pandas_enabled_datasource_config: dict,
+    datasource_config_with_names: DatasourceConfig,
```

Comment:
The actual datasource config is not important for this test, as long as it is a valid config.

Reply:
Could we add this to the docstring?

Reply:
Done!